pkgs/applications/networking/remote/x2goserver/default.nix

```diff
@@ -0,0 +1,66 @@
+{ stdenv, lib, fetchurl, makeWrapper, perl, perlPackages, which, nx-libs
+, utillinux, coreutils, glibc, gawk, gnused, gnugrep, findutils, xorg
+, nettools, iproute, bc, procps, psmisc, lsof, pwgen, openssh, sshfs
+}:
+
+let binaryDeps = [
+  perl which nx-libs utillinux coreutils glibc.bin gawk gnused gnugrep
+  findutils nettools iproute bc procps psmisc lsof pwgen openssh sshfs
+  xorg.xauth xorg.xinit xorg.xrandr xorg.xmodmap xorg.xwininfo xorg.fontutil
+  xorg.xkbcomp xorg.setxkbmap
+]; in
+stdenv.mkDerivation rec {
+  name = "x2goserver-${version}";
+  version = "4.1.0.3";
+
+  src = fetchurl {
+    url = "http://code.x2go.org/releases/source/x2goserver/${name}.tar.gz";
+    sha256 = "1l6wd708kbipib4ldprfiihqmj4895nifg0bkws4x97majislxk7";
+  };
+
+  buildInputs = binaryDeps;
```

Comment:
Are none of these `propagatedBuildInputs`?

Reply:
I don't quite know how to find out. If that just means probable runtime dependencies, then I think pretty much all of them. Should I move them to `propagatedBuildInputs`?

Reply:
`propagatedBuildInputs` isn't for runtime dependencies, and neither is `buildInputs`. If a binary needs some binaries you need to wrap its `PATH` with them, which you seem to be doing already, so there shouldn't be any need to add these to `buildInputs` at all.

Reply:
I needed perl + bash for patchShebangs, the rest is gone, and it seems to work.